resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -27,6 +27,24 @@ import org.apache.spark.network.util.ByteUnit
 package object config extends Logging {
 
   /* Common app configuration. */
+  private[spark] val SCHEDULING_REQUEST_ENABLED =
+    ConfigBuilder("spark.yarn.schedulingRequestEnabled")
```

Comment:
I think we need to  come up with better name  and more description.   From just reading this, I have no idea what it does.  spark always requests containers from yarn to schedule on, so why is it off...(ie that is what I read from the config name)